pkgs/development/python-modules/online-judge-tools/default.nix

```
@@ -0,0 +1,27 @@
+{ buildPythonPackage, colorama, fetchFromGitHub, lib, online-judge-api-client
+, requests, ... }:
+
+buildPythonPackage rec {
+  pname = "online-judge-tools";
+  version = "11.5.1";
+
+  src = fetchFromGitHub {
+    owner = "online-judge-tools";
+    repo = "oj";
+    rev = "v${version}";
+    sha256 = "0zkzmmjgjb6lyrzq1ip54cpnp7al9a7mcyjyi5vx58bvnx3q0c6m";
+  };
+
+  propagatedBuildInputs = [ colorama online-judge-api-client requests ];
+
+  # Requires internet access
+  doCheck = false;
+
+  meta = with lib; {
+    description =
+      "Tools for various online judges. Downloading sample cases, generating additional test cases, testing your code, and submitting it.";
```

```suggestion
    description = "Tools for various online judges. Download sample cases, generat additional test cases, test your code, and submit it.";
```
I am not sure what is proper English grammar but I think before it was a bit off.